Sources/NIO/ByteBuffer-aux.swift

```diff
@@ -373,6 +373,34 @@ extension ByteBuffer {
         self._moveWriterIndex(forwardBy: written)
         return written
     }
+    
+    /// Writes `byte` `repeated` times. Moves the writer index forward by the number of bytes written.
+    ///
+    /// - parameter byte: The `UInt8` byte to repeat.
+    /// - parameter count: How many times to repeat the given `byte`
+    /// - returns: How many bytes were written.
+    @discardableResult
+    public mutating func writeRepeatingByte(_ byte: UInt8, count: Int) -> Int {
+        precondition(count >= 0, "Can't write less than 0 bytes")
```

Comment:
I mean, it couldn’t possibly matter less, but...

```suggestion
        precondition(count >= 0, "Can't write fewer than 0 bytes")
```